addons/crm/crm_lead.py

```diff
@@ -730,28 +730,35 @@ def convert_opportunity(self, cr, uid, ids, partner_id, user_ids=False, section_
 
         return True
 
+    @api.one
```

Comment:
Both points are exactly what I want. The point of decorators is being able to mix both APIs. Just check it and you'll see it works.